openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1846,6 +1846,9 @@ The following additional Credential Issuer metadata are defined for this Credent
       * `display`: OPTIONAL. An array of objects, where each object contains display properties of a certain claim in the Credential for a certain language. Below is a non-exhaustive list of valid parameters that MAY be included:
           * `name`: OPTIONAL. String value of a display name for the claim.
           * `locale`: OPTIONAL. String value that identifies language of this object represented as language tag values defined in BCP47 [@!RFC5646]. There MUST be only one object for each language identifier.
+          * `logo`: OPTIONAL. A JSON object with information about the logo of the Credential Issuer with a following non-exhaustive list of parameters that MAY be included:
+            * `uri`: OPTIONAL. String value that contains a URI where the Wallet can obtain a logo of the Credential from the Credential Issuer. Wallet needs to determine the scheme, since the URI value could use `https:` scheme, `data:` scheme, etc.
+            * `alt_text`: OPTIONAL. String value of an alternative text of a logo image.
```

Comment:
Please delete this. This is new and unrelated to the #43. If a mechanism for the issuer logo is needed, then this should not be defined format-specific.
```suggestion
```

Reply:
aaaa good catch. I added this in the wrong place.

Reply:
you are right. this was born out of Paul's comment on this PR, and should be in another PR. i removed this from this PR and moved to PR #170